src/styles/blocks/card.css

```diff
@@ -0,0 +1,98 @@
+.card {
+  box-sizing: content-box;
```

Comment:
You've already set   * { 
box-sizing: border-box;
}
Do you need 'content-box' here?

Reply:
agree, you can remove it from here, to properly set sizes
```suggestion
```